bcachefs/bcachefs.py

```diff
@@ -170,6 +182,9 @@ def _parse(self):
         for parent_inode, ls in self._inodes_ls.items():
             self._inodes_ls[parent_inode] = self._unique_dirent_list(ls)
 
+        for inode in BcachefsIterInode(self._filesystem):
+            self._inode_map[inode.inode] = inode.size
+
```

Comment:
Maybe put the sanitizing step after the initialization of the fields
```suggestion
        for inode in BcachefsIterInode(self._filesystem):
            self._inode_map[inode.inode] = inode.size

        for parent_inode, ls in self._inodes_ls.items():
            self._inodes_ls[parent_inode] = self._unique_dirent_list(ls)

```